api/contract_test.go

```diff
@@ -84,3 +84,61 @@ func TestCreateFiltersRequestValid(t *testing.T) {
 		})
 	})
 }
+
+func TestCreateFilterOutputsRequestValid(t *testing.T) {
+
+	Convey("Given a valid createFilterOutputsRequest request object", t, func() {
```

Comment:
this doesn't fully test the code under test. Is that ok?

Reply:
We are just trying to test simple validation function, which is currently testing if any of the field is blank. Rest of the functionality is being tested in functional test